### PR TITLE
Add uid value to alert notification resource

### PR DESCRIPTION
**Pull request by 58231**
This is verified working. However, It does required the following PR be completed: https://github.com/nytm/go-grafana-api/pull/34

### CODE REVIEW

**Comment by lijok** [comment]
Any chance to get this merged in asap?
Impossible to set up alarms on panels with terraform without this

**Review by tonglil — APPROVED** [review]
Reviewing this, would you be able to resolve the conflict?

Otherwise LGTM, pending upstream library changes

**Comment by 58231** [comment]
Once a release gets cut for github.com/nytm/go-grafana-api (PR just got merged today), I'll get this PR updated and it should clear the conflict.

**Comment by mlclmj** [comment]
Hey @58231, just released [v0.5.0](https://github.com/nytm/go-grafana-api/releases/tag/v0.5.0) which includes these changes

File: grafana/resource_alert_notification.go [hunk 1]
```diff
@@ -38,6 +38,12 @@ func ResourceAlertNotification() *schema.Resource {
 				Optional:  true,
 				Sensitive: true,
 			},
+
+			"uid": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Required: false,
```
Comment (reugn) [inline grafana/resource_alert_notification.go]: You don't need Required next to Optional here. Replace it with `Computed: true`.

**Review by reugn — CHANGES_REQUESTED** [review]

File: grafana/resource_alert_notification.go (hunk 1 above)
Comment (58231) [reply grafana/resource_alert_notification.go]: Thanks! Changes made.

**Review by tonglil — APPROVED** [review]
Thanks!

**Pull request merged**